src/core/component.test.ts

```diff
@@ -1,5 +1,5 @@
 import Vue from "vue";
-import BaseComponent from "../core/component";
+import { DxComponent as BaseComponent, DxExtensionComponent} from "../core/component";
```

Comment:
A bit confusing to use `BaseComponent` along with `DxExtensionComponent`. As for me better use `DxComponent` and  `DxExtensionComponent` or `BaseComponent` and `ExtensionComponent`